api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -500,7 +500,7 @@ private void setInstitutionTierRequirement(
   // Take first 76 characters from display Name (with no spaces) and append 3 random number
   private String generateShortName(String displayName) {
     Random r = new Random();
-    displayName = displayName.replaceAll("\\s", "");
+    displayName = displayName.replaceAll("\\.", "").replaceAll("\\s", "");
```

Comment:
I think it will be more future-proof to use an "allowlist"-like approach, rather than a "denylist" like this.

Perhaps just: `replaceAll("[^a-zA-Z0-9]", "")`, i.e. only allow alphanumeric